dojo/tools/twistlock/parser.py

```diff
@@ -235,6 +237,10 @@ def get_item(vulnerability, test, image_metadata=""):
     status = vulnerability.get("status", "There seems to be no fix yet. Please check description field.")
     cvssv3_score = vulnerability.get("cvss")
     riskFactors = vulnerability.get("riskFactors", "No risk factors.")
+    try:
+        date = str(dateutil.parser.parse(vulnerability.get("discoveredDate")).date())
+    except (ValueError, TypeError, dateutil.parser.ParserError):
+        date = datetime.now()
```

Comment:
```suggestion
        date = timezone.now()
```
using timezone aware dates will suppress some warnings from the logs